src/styles/card.scss

```
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+.cards {
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 488px) {
+  .cards {
+    --column-count: 1;
+  }
+}
+
+@media (min-width: 488px) {
+  .cards {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    --column-count: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 36px;
+  }
+
+  &__name {
+    color: $Main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
```

also reset default margin for such tags as p, h2 etc
```suggestion
    margin-bottom: 4px;
```